StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentMethodType.swift

```diff
@@ -126,6 +126,19 @@ extension PaymentSheet {
                 }
             }
 
+            if
+                recommendedStripePaymentMethodTypes.contains(.link),
```

Comment:
this was changed from `intent.supportsLink`

which is:

```
extension Intent {
    var supportsLink: Bool {
        // Either Link is an allowed Payment Method in the elements/sessions response, or passthrough mode (Link as a Card PM) is allowed
        return recommendedPaymentMethodTypes.contains(.link) || linkPassthroughModeEnabled
    }
```

so the totality of the change is just removing `linkPassthroughModeEnabled`

which better matches what Android is doing

Reply:
Sweet!

Reply:
I feel like this should only be applicable if linkPassthroughMode is false?  Like, passthrough mode means merchants see the PM type as the underlying PM (so, US Bank Account here)?

Reply:
> I feel like this should only be applicable if linkPassthroughMode is false? Like, passthrough mode means merchants see the PM type as the underlying PM (so, US Bank Account here)?

I reached out to folks on this part - atm I believe Android doesn't contain this logic

Reply:
@yuki-stripe 

Talked to Android folks, and this looks to be fine as Instant Debits logic makes sure its false anyway (note that `intent.linkFundingSources?.contains(.bankAccount) == true` has a lot of logic behind it for it to be true [including some instant debits flags that must be true for it to add the instant debits payment method])

Quotes from @.carlosmuvi :

> passthrough mode cannot be enabled if the link payment method is passed (PM mode and PT mode are mutually exclusive).

> Instant debits is not compatible with passthrough mode (something that will change with Panther)